### PR TITLE
Wiho/task ¤

### PR DESCRIPTION
**Pull request by WilliamHo1999**
Task 3, 4 and 5.

Please look for spelling mistakes and stuff.

IMPORTANT: Also, when you guys look at the pull requests, please make sure that I've written all loss functions and metrics (r2/accuracy) in the form of:

function(prediction, true)

Also, im malding over not using the correct git account...

### CODE REVIEW

File: Project2/csnet/metrics.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+
+def accuracy(pred: np.ndarray, true: np.ndarray) -> int:
```
Comment (tomrtk) [inline Project2/csnet/metrics.py:4]:
```suggestion
def accuracy(pred: np.ndarray, true: np.ndarray) -> float:
```

File: Project2/task3.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    # Scale data by subtracting mean
+    scaler_input = StandardScaler(with_std = False)
+    scaler_input.fit(X_train)
+    scaler_output = StandardScaler(with_std = False)
+    scaler_output.fit(z_train)
+
+    X_train = scaler_input.transform(X_train)
+    X_eval = scaler_input.transform(X_eval)
+    X_test = scaler_input.transform(X_test)
```
Comment (tomrtk) [inline Project2/task3.py:40]: Should test data not be scaled by it self and not based on training data?
